wiki/Documentation-single-source-of-truth.md

```diff
@@ -119,22 +127,22 @@ Hierbei ändert sich nichts im Vergleich zu Daniels Aufschlag einer SSoT. Nutzer
 }
 ```
 
-**BPMN benötigt:**
-- id :heavy_check_mark: 
-- Label -> name :heavy_check_mark: 
-- Vorgänger :heavy_check_mark: 
-- Nachfolger :heavy_check_mark: 
-- ~~Position~~ -> berechnet :heavy_check_mark: 
-- ~~Größe~~ -> `config.json` :heavy_check_mark: 
-- _RPA-Task & Application & Parameters_ :heavy_check_mark: 
+**BPMN requires:**
+- id :heavy_check_mark:
+- Label -> name :heavy_check_mark:
```

Comment:
We should decide for all the bullet points (this also goes for all the following bullet points) whether each bullet point starts with a capital letter or not. It should be consistent though.